---
templateKey: old
title: Project 2019
teamYear: 2019
---
### 

<html>

<div class="column full_size">

<h1>Immobilizing antibodies in correct orientation </h1>

<h2>Expanding genetic code</h2>

<p>Immunoassays are widely used tools to help diagnosing and managing diseases as well as detecting toxins in the environment. One of the most known example of such tool is pregnancy test which detects human chorionic gonadotropin, a hormone produced by the placenta.</p>

<p>Immunoassays are based on the specific detection ability of antibodies. Usually an immobilized capture antibody is used to bind an antigen of interest onto a surface, where a dye labeled antibody binds to the same analyte allowing the visual detection of it. However using traditional technologies to immobilize the binding antibodies leads to a “mess”, where the antibodies are randomly attached to the test surface hindering their ability to bind the antigens. <a href="https://avs.scitation.org/doi/10.1116/1.4978435"> 1 </a> (Fig 1)

</p>

<img src="https://2019.igem.org/wiki/images/8/8d/T--Aboa--mess_of_antibodies_Des_060619.jpeg" style="width:285px;height:197px;">

<p><i><b>Figure 1.</b> Randomly immobilized binder antibodies on a test surface. When the binder antibodies are immobilized randomly not all of them can bind their analytes because the paratopes are blocked. Modified from <a href="https://avs.scitation.org/doi/10.1116/1.4978435"> 1 </a>.</i> </p>

</div>

<div class="column two_thirds_size" >

<p>To increase the sensitivity of the immunoassays we are going to coat a test surface with as many correctly positioned antibodies as possible and this way increase the amount of bound antigens. In order to achieve this we are going to site-specifically modify the binder antibodies using expanded genetic code to incorporate a noncanonical amino acid (ncAA) into the tail of the antibody <a href="https://www.nature.com/articles/nature24031"> 2 </a>. </p>

<p> To add an additional amino acid into the antibody, we are going to utilize the amber stop codon (TAG or UAG) by adding it into the sequence of the antibody, and expressing that in a host that has all its amber codons replaced with other stop codons in its whole genome. The ncAA that we are going to insert into the antibody is p-Azido-L-phenylalanine, which is known to rapidly react with an alkyne group forming a covalent bond <a href=https://pubs.rsc.org/en/content/articlelanding/2013/AN/c2an36787d#!divAbstract"> 3 </a>. (Fig 2) Because this reaction is also bio-orthogonal, it does not occur between any natural groups present in the protein or the organism <a href=https://pubs.acs.org/doi/10.1021/cb4009292"> 4 </a>. </p>

<img src="https://2019.igem.org/wiki/images/9/98/T--Aboa--oriented_antibodies_Des_060619.jpeg" style="width:285px;height:197px;">

<p><i><b>Figure 2.</b> Site specifically orientated binder antibodies. Because the azide group of the p-azido-L-phenylalanine reacts bio-orthogonally with an alkyne group forming a covalent bond, the antibody with the amino acid in its tail is forced in the correct position allowing the paratopes to interact with the analytes. Modified from figure <a href=https://pubs.rsc.org/en/content/articlelanding/2013/AN/c2an36787d#!divAbstract"> 3 </a>.</i></p>

</div>

<div class="info box">

<h3>Expanded genetic code</h3>

<p>Expanded genetic code is a technique where one of an organism’s codons, for example a stop-codon, is reprogrammed to code for a ncAA <a href="https://www.nature.com/articles/nature24031"> 2 </a>.</p>

<p>This includes replacing a desired codon throughout the genome with an analogous codon and adding an aminoacyl-tRNA synthetase recognizing the replaced codon to be able to introduce a ncAA instead <a href="https://www.nature.com/articles/nature24031"> 2 </a>.</p>

</div>

<div class="clear extra_space"> 

<div class="column full_size">

<h2>Inspiration behind the idea</h2>

<p>The inspiration to our work came from reading about the latest advances in genetic code expansion. The Department of Biotechnology in Turku is heavily concentrated on diagnostic development and therefore when our students were reading about expanded genetic code and the possibilities it provides in the field of site specific conjugations, it really did not take too long to come up with a simple and minimalistic, yet powerful idea of site-specific conjugation of antibodies for the use of immunoassays. This technology has already been used for immobilization of proteins <a href=https://pdfs.semanticscholar.org/a414/d2e600924dc685c3acdb7ab91e2bfa06fb31.pdf"> 5 </a> and modication of therapeutic  antibodies <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6190509/"> 6 </a>. So why not combine these two in a very intuitive way?</p>

<p>The basic principle of using antibodies in diagnostic tools has been a cornerstone of the whole industry for decades now but the whole concept of antibody immobilization is a stochastic process which in turn means that basically the physicochemical model of an immunoassay is at its best, a good guess.</p> 

<p>Therefore the optimal performance of any immunoassay could be reached with a minimalistic approach that will orient the antibodies correctly without using any other proteins such as streptavidin to bind antibodies into a sensor surface. This approach could be achieved by using two extremely convenient techniques: click-chemistry and genetic code expansion. This would lead to a more defined biosensors and open up new possibilities in optimizing them to suit the future needs of diagnostics <a href="https://avs.scitation.org/doi/10.1116/1.4978435"> 1 </a>.</p>

</div>

</div>

<div class="column full_size">

<h3>References</h3>

<p>1. Welch, N.G., Judith, A.S., Muir, B.W. & Pigram, P.J. (2017) Orientation and characterization of immobilized antibodies for improved immunoassays (Review). Biointerphases doi: 10.1116/1.4978435 </p>

<p>2. Chin, J.W. (2017) Expanding and reprogramming the genetic. Nature <b>550:</b> 53-60. </p>

<p>3. Trilling, A., Beekwilder, J. & Zuihof, H. (2013) Antibody orientation on biosensor surfaces: a minireviews. The Analyst <b>138:</b> 1619-1627. </p>

<p>4. Lang, K. & Chin, J. (2014) Bioorthogonal Reactions for Labeling Proteinss. ACS Chemical Biology <b>9:</b> 16-20. </p>

<p>5. Raliski, B., Howard, C. & Young, D. (2014) Site-Specific Protein Immobilization Using Unnatural Amino Acids. Bioconjugate Chemistry <b>25:</b> 1916-1920. </p>

<p>6. Huang, Y. & Liu, T. (2018) Therapeutic applications of genetic code expansion. Synthetic and Systems Biotechnology <b>3:</b> 150-158. </p>

</div>

</html>
